README.md

```diff
@@ -1,13 +1,36 @@
 # MP-DS
 Master Project Distributed System
 
```

Comment:
@pfisterb Could you add a small description how to reconfigure the Grafana dashboard regarding the metrics once the cluster and tools were set up again?